main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
+from django.utils import timezone
+import datetime
+
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_name, player_data in players_data.items():
+        race_data = player_data.get("race")
+        if race_data:
+            race, created = Race.objects.get_or_create(
```

Comment:
There is no need to allocate a separate variable `created`, it is not used for logic in other places. Omit the value using `_`.